### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by FabioAntunes**

### CODE REVIEW

**Comment by laughedelic** [comment]
Cool :+1:
Could you add the link to the Portuguese translation to other translations as well?

**Comment by FabioAntunes** [comment]
@bucaran  are you ok with merging this?

File: docs/ru-RU/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [travis-link]: https://travis-ci.org/fisherman/fisherman
 [travis-badge]: https://img.shields.io/travis/fisherman/fisherman.svg
 
-[fish shell]: https://github.com/fish-shell/fish-shell
+[fish-shell]: https://github.com/fish-shell/fish-shell
```
Comment (laughedelic) [inline docs/ru-RU/README.md]: This breaks the link in the first heading. Could you remove this dash? Or add it in the heading link. I don't know what is the canonical form to refer to fish..

Comment (bucaran) [inline docs/ru-RU/README.md]: Best way should be with a dash.

**Comment by bucaran** [comment]
@FabioAntunes Yes, please go ahead a merge yourself after fixing the points indicated by @laughedelic.

**Comment by fabio-flubit** [comment]
@laughedelic changed the title to the dashed one, are you ok with merging?

**Comment by FabioAntunes** [comment]
@laughedelic changed the title to the dashed one, are you ok with merging?

**Comment by bucaran** [comment]
LGTM

**Pull request merged**